account_credit_control/migrations/12.0.1.6.1/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """
+    Convert custom_text fields to Html on policy_level
+    """
+    _logger.info(">> Post-Migration 12.0.1.6.1")
+    cr = env.cr
+    openupgrade.convert_field_to_html(
+        cr, "credit_control_policy_level", "custom_text", "custom_text"
```

Comment:
Please do a copy of the column first.

Reply:
What would be the purpose of this copied column ?
Is it just to keep a text version of the data in DB in case we want to convert it back ?

Reply:
Yes, it's a good migration practice to keep previous data unaltered (we do it constantly on OpenUpgrade), in case you need to compare, revert back, re-apply...

I forgot to mention that you have other openupgradelib method that eases this and you can use `None, None` for the rest of the arguments, sorry:

https://github.com/OCA/openupgradelib/blob/6c18ce0c160b9cf47c4fa898d9b6c4265d98c216/openupgradelib/openupgrade.py#L348

Reply:
Ok I understand, thank you.